client/pages/venue.jsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 
 const Venue = () => {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
I don't think this comment is necessary.

Reply:
It's necessary, as it underlines the line underneath, which doesn't allow me to commit.

Reply:
then you can delete the variable you are not using 😄